src/core/utils/mixedStorage.cpp

```diff
@@ -19,6 +19,7 @@ using namespace forte::core::util;
 
 void* CMixedStorage::write(const void *paBegin, std::size_t paSize) {
   auto pBegin = reinterpret_cast<const char*>(paBegin);
+  const std::size_t offset = mData.size();
```

Comment:
Is it really necessary to specify std namespace here for size_t?

Reply:
It all comes from the initial of implementation.